src/mugs.js

```diff
@@ -356,7 +356,8 @@ define([
                 .replace("$2", property)
                 .replace("$3", String(vis)));
         },
-        getDisplayName: function (lang) {
+        getDisplayName: function (lang, escape) {
+            if (escape === undefined) { escape = true; }
```

Comment:
yes. I read up on it and `x === undefined` should be safe. I've stopped using `_.isUndefined` because I think it's ugly.